### PR TITLE
Test commit 1.

### PR DESCRIPTION
**Pull request by chrismmillar**
Hello.

### CODE REVIEW

File: _slides/06-chrismmillar.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+TEST!!!
```
Comment (vvaths) [inline _slides/06-chrismmillar.md:6]:
```suggestion
TEST COMMENT!!!
```

File: _slides/06-chrismmillar.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vvaths) [inline _slides/06-chrismmillar.md:5]: looks great!

**Review by vvaths — COMMENTED** [review]
Good job!

**Review by dgahimer89 — APPROVED** [review]
Very cool.

**Review by vvaths — APPROVED** [review]
ELGTM!

**Pull request merged**